mozilla-release/mobile/android/app/build.gradle

```diff
@@ -303,6 +303,14 @@ dependencies {
 
     // Including the Robotium JAR directly can cause issues with dexing.
     androidTestCompile 'com.jayway.android.robotium:robotium-solo:5.5.4'
+
+    /* Cliqz start */
+    // add butter knife for view binding
+    compile ('com.jakewharton:butterknife:8.8.1') {
```

Comment:
We have already a ton of dependencies, I think we have to wave butterknife and do it manually

Reply:
I think it's better to keep it. our layout is fully depending on it.
and it is much easier while coding.

Reply:
Let me rephrase this: do it manually